cove/input/models.py

```diff
@@ -29,6 +29,9 @@ class SuppliedData(models.Model):
     modified = models.DateTimeField(auto_now=True, null=True)
     rendered = models.BooleanField(default=False)
 
+    # Last schema version applied to the stored data
+    schema_version = models.CharField(max_length=10, null=True, default='')
```

Comment:
Not sure why this has null=True, and defaults to the empty string. Could we have default=None?